### ISSUE TITLE
Strange behavior creating new project (V9)

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord-dev/ts-desktop#660]
After discovering that 9.0-09 had the 8.1 app code, I uninstalled it. I did not uninstall Git. I then installed the build labeled 9.0-7. I then uninstalled Git and downloaded and installed the latest version. Same behavior. I then renamed my targetTranslations folder to start from scratch. The Updating project status now is somewhat faster, but still pretty slow. The extra target language selection screen is not shown now with just the one new project. On 9.0-11 this is still a problem. It is showing the extra target language screen consistently once I have several projects defined.

[translationStudio 6_14_2016 12_00_39 PM.zip](https://github.com/unfoldingWord-dev/ts-desktop/files/314537/translationStudio.6_14_2016.12_00_39.PM.zip)
This recording shows what happens when creating a new OBS project. After selecting the target language, the dialog to select the target language comes up again. When the status shows Üpdating project list (which seems to take a long time), everything else seems to be blocked or at least disrupted. After closing the project, I am unable to close the app by clicking the X until after the Updating project list has completed.

**Comment by cckozie** [comment]
This appears to be better in 9.0-11, but it is still not completely fixed. It does still appear, but the extra select target language screen does stay visible as long as it did before. It also appears that how it behaves is dependent on how large the project is.

**Comment by cdwhitfield73** [comment]
This should be resolved now.  The two things that were happening during the "Update List" process that were taking a long time was git committing all the projects and auto backing up all the projects.  I changed it so these two functions only happen on start up, during the initial Updating/Creating of the project list.  Any time the list needs updated during use, such as creating a new project or making changes to a project, or importing a new project, the project in question is git committed separately and the update list function just refreshes the list (which is quick) without automatically committing and backing up all the projects.  I also changed where the new project menu "resets" itself to the language page so this will not happen at the end of the process while it is creating the new project, so the user will never see it switch.  I also reduced the busy signal pause for all functions to half a second from a full second to speed up the time it takes to perform multiple operations.
